spec/spec.md

```diff
@@ -29,11 +29,15 @@ Participate:
 
 ## Abstract
 
-The JWT VC Presentation Profile defines a set of specifications to enable the interoperable presentation of Verifiable Credentials between Wallets and Verifiers/RPs.
+The JWT VC Presentation Profile defines a set of specifications to enable the interoperable presentation of [[ref:Verifiable Credentials]] (VCs) between digital [[ref: Wallets]] and credential [[ref: Verifiers]].
```

Comment:
let's just say "between wallet and verifiers", or redefine terminology to "digital wallet", and "credential verifier"?

Reply:
can do!